src/main/java/org/dbflute/intro/app/logic/dfprop/DfpropInfoLogic.java

```diff
@@ -241,6 +244,24 @@ private SchemaPolicyColumnMap parseColumnMap(Map<String, Object> schemaPolicyMap
         return new SchemaPolicyColumnMap(themeList, originalStatementList);
     }
 
+    public List<String> getStateMentSubjectList() {
+        // 頻繁に変わる内容ではないのでベタ書き実装している。
+        ArrayList<String> subjectList = new ArrayList<String>();
```

Comment:
Consider making it an Enum. 🤔

Reply:
@EgumaYuto 
I created an enum.
9a658b34c7353d99043dd4beaee437b66fbefe45